### PR TITLE
Add landing Page

### PR DESCRIPTION
**Pull request by hassanmasood007**
1. Added  Appbar and Drawer on Landing page
2. Created pages for Every item in Drawer

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import Home from "./containers/Home";
+import Notification from "./containers/Notification";
+
+
+
+const App:FunctionComponent = () => {
+  return (
+    <div>
+      <BrowserRouter>
+      <Navbar />
```
Comment (MuhammadFaizan19) [inline src/App.tsx:18]: as said above should come from a layout

File: src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <Switch>
+          <Route path={" About, Feed, Profile, Signup, Login, Notification , /"} />
```
Comment (MuhammadFaizan19) [inline src/App.tsx:20]: This Route is a self closing tag which is not right, this one should be the parent to every other Route tag inside this Switch and also the Layout component should also be in this Route tag

File: src/components/Navbar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
+import { styled, useTheme } from '@mui/material/styles';
+import Box from '@mui/material/Box';
```
Comment (MuhammadFaizan19) [inline src/components/Navbar/index.tsx:2]: everything coming from '@mui/material' should all be in a single import statement e.g import {Box, Typography} from '@mui/material'

File: src/components/Navbar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+import Divider from '@mui/material/Divider';
+import IconButton from '@mui/material/IconButton';
+import MenuIcon from "@material-ui/icons/Menu";
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import { ChevronLeft, ChevronRight } from '@material-ui/icons';
+
+
+import { useHistory } from 'react-router';
+import { FunctionComponent, useState } from 'react';
```
Comment (MuhammadFaizan19) [inline src/components/Navbar/index.tsx:18]: Import statements from packages should be on top of the file

**Review by MuhammadFaizan19 — CHANGES_REQUESTED** [review]

**Comment by MuhammadFaizan19** [comment]
Also add a linter and have a command in package.json there should be two commands, 'npm run lint' and 'npm run lint:fix' ,might as well add 'prettier' for formatting too.

File: src/App.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+import { FunctionComponent } from "react";
+import Navbar from "./components/Navbar";
```
[the remark this answers is not in the document]
Comment (hassanmasood007) [reply src/App.tsx:3]: Thanks for your review. Working on it

File: src/App.tsx (hunk 1 above)
Comment (hassanmasood007) [reply src/App.tsx:18]: Thanks for your review. Working on it

File: src/components/Navbar/index.tsx (hunk 4 above)
Comment (hassanmasood007) [reply src/components/Navbar/index.tsx:18]: Thanks for your review. Working on it

File: src/App.tsx (hunk 2 above)
Comment (hassanmasood007) [reply src/App.tsx:20]: Thanks for your review. Working on it

File: src/App.tsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+import Home from "./containers/Home";
+import Notification from "./containers/Notification";
+
+const App: FunctionComponent = () => {
+  return (
+    <div>
+      <BrowserRouter>
+        <Switch>
+          <Route
+            path={[
```
Comment (MuhammadFaizan19) [inline src/App.tsx:19]: Add exact property on this

File: src/App.tsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+              "About",
+              "Feed",
+              "Profile",
+              "Signup",
+              "Login",
+              "Notification",
+              "/",
+            ]}
+          >
+            <HomeLayout />
```
Comment (MuhammadFaizan19) [inline src/App.tsx:29]: make it an higher order component

**Review by MuhammadFaizan19 — CHANGES_REQUESTED** [review]

**Review by MuhammadFaizan19 — APPROVED** [review]

**Pull request merged**